specification/communication/resource-manager/Microsoft.Communication/preview/2023-04-01-preview/CommunicationServices.json

```diff
@@ -686,7 +686,12 @@
           "x-ms-client-flatten": true
```

Comment:
2020-08-20-preview has gone through the full deprecation announcement process and RP and ARM no longer accepts requests for this API Version